package.json

```diff
@@ -1,21 +1,35 @@
 {
-  "name": "orchardcore",
+  "name": "@orchardcore/root",
   "private": true,
+  "version": "1.0.0",
+  "packageManager": "yarn@4.5.3",
   "scripts": {
-    "build": "gulp build",
-    "rebuild": "gulp rebuild",
-    "watch": " gulp watch",
-    "postinstall": "node .scripts/install.js",
+    "gulp-build": "gulp build",
+    "gulp-watch": "gulp watch",
+    "gulp-rebuild": "gulp rebuild",
     "update-projects": "node .scripts/update.js",
     "install-project": "node .scripts/project.js install",
     "update-project": "node .scripts/project.js update",
-    "cleanup": "node .scripts/cleanup.js"
+    "cleanup": "node .scripts/cleanup.js",
+    "build": "asset-bundler build",
+    "copy": "asset-bundler copy",
+    "watch": "asset-bundler watch",
```

Comment:
Should this work already?

![image](https://github.com/user-attachments/assets/99248d21-92f1-47fe-8cad-c8390dae8220)

Reply:
The issue is that I'm not providing any Asset2.json bundler configuration yet.
Will need to add a null check there so that it doesn't show this error message.
I will add an example.